### PR TITLE
Resources

### PR DESCRIPTION
**Pull request by YesicaAcha**

### CODE REVIEW

File: src/test/java/org/example/schoology/pages/resources/MyResources.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    }
+
+    public String getMessage() {
+        wait.until(ExpectedConditions.visibilityOf(messageText));
+        return messageText.getText();
+    }
+
+    public boolean isResourceDisplayed(final String resourceName) {
+        try {
+//			// Changing timeout
```
Comment (aarteagavargas) [inline src/test/java/org/example/schoology/pages/resources/MyResources.java]: there is a duplicated "//"

File: src/test/java/org/example/schoology/pages/resources/MyResources.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            driver.manage().timeouts().implicitlyWait(MIN_IMPLICIT_TIMEOUT, TimeUnit.SECONDS);
+            driver.findElement(By.xpath(String.format(RESOURCES_ACTIONS_BUTTON, resourceName)));
+            return true;
+        } catch (NoSuchElementException e) {
+            // nothing.
+        } finally {
+//			// Restore timeout
```
Comment (aarteagavargas) [inline src/test/java/org/example/schoology/pages/resources/MyResources.java]: there is a duplicated "//"

File: src/test/java/org/example/schoology/pages/courses/Courses.java [hunk 3]
```diff
@@ -18,4 +26,17 @@ public String getCourseTitle(final String courseName) {
...
         return action.getText(By.xpath(xPath));
     }
+
+
+    public DeletePopup clickDeleteCourse(String courseName) {
+        WebElement courseActionsButton = driver.findElement(By.xpath(String.format(COURSE_ACTIONS_BUTTON, courseName)));
+
+        // Scroll
+        JavascriptExecutor js = (JavascriptExecutor) driver;
+        js.executeScript("arguments[0].scrollIntoView();", courseActionsButton);
```
Comment (aarteagavargas) [inline src/test/java/org/example/schoology/pages/courses/Courses.java]: use the method created in common actions

File: src/test/java/org/example/schoology/pages/resources/MyResources.java (hunk 1 above)
Comment (YesicaAcha) [reply src/test/java/org/example/schoology/pages/resources/MyResources.java]: the method was not used, I removed it

File: src/test/java/org/example/schoology/pages/courses/Courses.java (hunk 3 above)
Comment (YesicaAcha) [reply src/test/java/org/example/schoology/pages/courses/Courses.java]: Done

File: src/test/java/org/example/schoology/pages/resources/MyResources.java (hunk 2 above)
Comment (YesicaAcha) [reply src/test/java/org/example/schoology/pages/resources/MyResources.java]: method removed

File: src/test/java/org/example/schoology/pages/courses/Courses.java (hunk 3 above)
Comment (YesicaAcha) [reply src/test/java/org/example/schoology/pages/courses/Courses.java]: done

**Review by aarteagavargas — APPROVED** [review]

**Pull request merged**